README.md

```diff
@@ -124,9 +131,19 @@ Here are all available options with their default values:
 
 @cpu_medium_thresh "30" # medium percentage threshold
 @cpu_high_thresh "80" # high percentage threshold
+
+@ram_(low_icon,high_bg_color,etc...) # same defaults as above
+
+@cpu_temp_format "%2.0f" # printf format to use to display temperature
+@cpu_temp_units "C" # supports C & F
+
+@cpu_temp_medium_thresh "80" # medium temperature threshold
+@cpu_temp_high_thresh "90" # high temperature threshold
```

Comment:
you need to change this also.

Reply:
Sorry, I don't see what needs changing.

Reply:
Do you mean there's no `@gpu_temp_high_thresh`? No, there isn't - it just uses the CPU threshold. For that matter, there's never been a `@gpu_high_thresh` either.

Reply:
Ok my bad I miss read the "variable rename" commit.